### ISSUE TITLE
Data Management Upgrade:

### ISSUE THREAD

**Issue by jamespatrickmanning** [GMaynard1/emolt_project_management#10]
Telemetry_status enhancement:

-[] "QUE" tab should have the same columns as "fitted"
-[] new columns needed for "company name", "owner contact", "fleet manager contact"
-[] ultimately this googlesheet should be loaded into a mySQL database

**Comment by GMaynard1** [comment]
Hey @jamespatrickmanning to make the check boxes / tasks appear, there need to be spaces between the hyphen and brackets, so it should look like
```
- [ ]
```

**Comment by ErinGOMLF** [comment]
Jim do we still need to do this or will the new database that George is creating going to make this not an issue?

**Comment by xhx509** [comment]
I will keep the google sheet Telemetry_status updated, add both visits of today, and yesterday.

**Comment by jamespatrickmanning** [comment]
Yes. We can close this issue once we have the new mySQL method to store "telemetry_status".  In the meantime, lets make sure telemetry_status is up to date.

**Comment by GMaynard1** [comment]
Yup, here's a few screenshots of what's in development, not ready yet
![log_visit](https://user-images.githubusercontent.com/77748586/169584293-ab405c60-f957-4abc-bb9b-0f07e56c2f78.PNG)
![confirmation](https://user-images.githubusercontent.com/77748586/169584317-ac9ad2c3-4464-4db1-bae6-04129f428614.PNG)
![confirmation2](https://user-images.githubusercontent.com/77748586/169584323-b627b355-74ac-46a4-8261-6840c97edb26.PNG)
![hardware_yn](https://user-images.githubusercontent.com/77748586/169584337-ca9c1b96-1229-4dde-b5c6-9d06373a7c4b.PNG)
![hardware_log](https://user-images.githubusercontent.com/77748586/169584350-2812b98e-e057-41d7-90fa-55784d9f6356.PNG)
![miss_julie](https://user-images.githubusercontent.com/77748586/169584360-182bba8b-6fdc-4a4a-8cb5-059a27078f87.PNG)

.
